### PR TITLE
Replaced PropertyMetadata nuget package

### PR DESCRIPTION
**Pull request by un-tone**
For https://github.com/StarcounterApps/Calendar/issues/34

### CODE REVIEW

File: src/Calendar/packages.config [hunk 1]
```diff
@@ -1,4 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Smorgasbord.PropertyMetadata" version="1.0.0.1" targetFramework="net45" />
-</packages>
```
Comment (warpech) [inline src/Calendar/packages.config]: If `packages.config` is removed, you can also remove usage of `nuget.exe` from `build.bat` and the file `tools\nuget.exe`.

Comment (warpech) [inline src/Calendar/packages.config]: Also, I have some problem with `build.bat` now. Can you pls check if it works for you?

2017-03-29 17_44_09-c__windows_system32_cmd exe

**Comment by un-tone** [comment]
Closing. New PR https://github.com/StarcounterApps/Calendar/pull/37.

File: src/Calendar/packages.config (hunk 1 above)
Comment (un-tone) [inline src/Calendar/packages.config]: I checked for this PR https://github.com/StarcounterApps/Calendar/pull/37. There was some compilling warnings which I fixed.

**Pull request closed without merging**